test/e2e/specs/specs-wpcom/wp-gutenberg-experimental-features-spec.js

```diff
@@ -22,7 +22,7 @@ const gutenbergUser =
 // of the @wordpress/* packages. The purpose of these tests is to give us an early
 // warning if an experimental feature has been removed or renamed.
 const EXPERIMENTAL_FEATURES = {
-	'@wordpress/block-editor': [ '__experimentalBlock', '__experimentalInserterMenuExtension' ],
+	'@wordpress/block-editor': [ '__experimentalBlock', '__unstableInserterMenuExtension' ],
```

Comment:
I just realized this change will probably work well for the purposes for this PR. The goal is to merge this ASAP in order to be able to install Gutenberg v10.7.0 RC1 on edge. However, it'll be a while until we're able to completely upgrade WPCOM to v10.7.x, and this test will be failing until then :(

I'll need to support both symbols here for now 🤔

Reply:
Or to make things simpler, I might just revert this for now and update it once we push v10.7.x to production. I'll add a reminder in the tracking issue for v10.7.x.

Reply:
Done, reverted and added a TODO item to the [tracking issue](https://github.com/Automattic/wp-calypso/issues/53135).